response_operations_ui/contexts/collection_exercise.py

```diff
@@ -242,7 +248,10 @@ def _build_ci_table(
             ci_details.append(_ci_details_item("eq", "EQ formtypes", ci_count, eq_url, ci_table_link_text))
 
             if app.config["CIR_ENABLED"]:
-                ci_details.append(_ci_details_item("cir", "CIR version", 0))
+                cir_count = get_cir_instrument_count(ce_id)["registry_instrument_count"]
```

Comment:
This isn't ideal, but the count is tucked inside the context builder

Reply:
I think this is fine